tests/test_integration_service.py

```diff
@@ -506,6 +506,8 @@ def test_orgunit(self):
                     'uuid': '32547559-cfc1-4d97-94c6-70b192eff825',
                 },
                 'parent': None,
+                'path': '',
```

Comment:
Any particular reason why root units don't have a path?

Reply:
The convention I have followed is that the name of the ou itself should not be part of the path. The name of the ou is written immediately above, and thus it would look silly, to repeat it + it keeps the path slightly shorter and thus avoids line-breaks in more cases. In this convention, the root ou will have no path.

Reply:
> The convention I have followed is that the name of the ou itself should not be part of the path. The name of the ou is written immediately above, and thus it would look silly, to repeat it + it keeps the path slightly shorter and thus avoids line-breaks in more cases. In this convention, the root ou will have no path.

Usually, the path to an item includes the name of that path; for example, a file system path _always_ includes the final item. I think they consider this a form of identification as well, in which case they almost certainly want to include the item itself.